Dockerfile

```diff
@@ -23,5 +23,5 @@ RUN yarn install
 
 COPY . /srv/app/
 
-EXPOSE 3000
-CMD ["rails", "server", "-b", "0.0.0.0", "-p", "3000"]
+EXPOSE 3001
+CMD ["rails", "server", "-b", "0.0.0.0", "-p", "3001"]
```

Comment:
I don't think we should change this port: since tapp and cp rails apps run in separate containers, using port 3000 on both sides would not conflict.